docs/nuget-overview.md

```diff
@@ -14,8 +14,10 @@ If you'd like to use a specific version, you can specify it like this:
 #r "nuget:System.Text.Json,4.7.2"
 ```
 
+## Adding a Nuget Source
 
+If your nuget package is not hosted on the main Nuget feed you can specify an alternative nuget source using `#i`.
 
-
-
-
+```csharp
+#i "https://www.myget.org/F/{feedurl}/api/v3/index.json"
```

Comment:
A few suggestions: 

* Using a real feed URL would be useful, to avoid people using this example literally.
* An example pointing out that this also supports file paths.
* The official syntax is `#i "nuget:...". While your example will work, there's some discussion over making it stricter.

Reply:
I addressed your feedback.

> An example pointing out that this also supports file paths.

I know you suggested to use a real example for the remote nuget source. Is it OK to use a fake example here for a local folder. It feels like this might be the most typical usage.